app/models/public_body.rb

```diff
@@ -668,12 +668,28 @@ def self.extract_domain_from_email(email)
     $1.nil? ? nil : $1.downcase
   end
 
-  def has_notes?
-    notes.present?
+  def notes
```

Comment:
This borks the table view. Might not be an issue depending on other PRs

![Screenshot 2022-08-30 at 11 42 12](https://user-images.githubusercontent.com/282788/187416821-09f0ecb2-4ea0-4761-8572-5562ef5d2464.png)

Reply:
This is addressed in #7245